src/components/Pages/Home.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
are these full pages or components? Robert: I feel that the Pages should be outside the components folder

Reply:
done, moved the pages folder outside of components